### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by harshalini**
Added Explore page in Video Library.
The videos are listed in the form of cards.

Preview -
https://gorgeous-entremet-bdf2b0.netlify.app/

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
This Code looks fine to me.
This PR is good to merge.

File: src/components/videos/videoList.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useVideo } from "../../contexts/videolist-context";
+export const VideoList = () => {
+    const { video } = useVideo();
+    return (
+        <div className="videos-grid">
+            {video.map((mp4) => (
```
Comment (rkapoor10) [inline src/components/videos/videoList.js]: you can destructure mp4

File: src/contexts/videolist-context.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const VideosProvider = ({ children }) => {
+  const [video, setVideo] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const {
+          data: { videos }
+        } = await axios.get("/api/videos");
+        setVideo(videos);
+      } catch {
```
Comment (rkapoor10) [inline src/contexts/videolist-context.js:14]: `catch(error)` this should be given

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge...a few comments added, pls go through them

File: src/components/videos/videoList.js (hunk 1 above)
Comment (harshalini) [reply src/components/videos/videoList.js]: Thanks, will do it now

**Pull request merged**